lib/Layout/Resolver/TargetHandler/Doctrine/SingleTaxon.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Layout\Resolver\TargetHandler\Doctrine;
+
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Types\Types;
+use Netgen\Layouts\Persistence\Doctrine\QueryHandler\TargetHandlerInterface;
+
+/**
+ * @deprecated this class will be renamed to simply Taxon in next major release
+ */
+final class SingleTaxon implements TargetHandlerInterface
+{
+    public function handleQuery(QueryBuilder $query, mixed $value): void
+    {
+        $query->andWhere(
+            $query->expr()->in('rt.value', [':target_value']),
```

Comment:
If this is a single taxon identifier, shouldn't this be `$query->expr()->eq(...)` ?

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/17/commits/057d41669cd3128b87eaf8063b07be5151bc5382